codalab/lib/codalab_manager.py

```diff
@@ -223,6 +223,11 @@ def init_config(self, dry_run=False):
             config['server']['engine_url'] = "sqlite:///{}".format(sqlite_db_path)
             print "Using SQLite database at: {}".format(sqlite_db_path)
 
+        # Generate secret key
+        config['server']['secret_key'] = get_random_string(
+            48, "=+/abcdefghijklmnopqrstuvwxyz"
+                "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789")
```

Comment:
-1 to feature requests not critical to migrating authentication as soon as
possible.

On Mar 2, 2016 10:47 PM, "Stephen Koo" <notifications@github.com> wrote:
>
> In codalab/lib/codalab_manager.py:
>
> > @@ -223,6 +223,11 @@ def init_config(self, dry_run=False):
> >              config['server']['engine_url'] =
"sqlite:///{}".format(sqlite_db_path)
> >              print "Using SQLite database at: {}".format(sqlite_db_path)
> >
> > +        # Generate secret key
> > +        config['server']['secret_key'] = get_random_string(
> > +            48, "=+/abcdefghijklmnopqrstuvwxyz"
> > +                "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789")
>
> I was considering that too. If that makes sense, we can go ahead and add
that.
>
> —
> Reply to this email directly or view it on GitHub.